internal/bminventory/inventory.go

```diff
@@ -4850,8 +4858,9 @@ func (b *bareMetalInventory) V2RegisterHost(ctx context.Context, params installe
 				log.WithError(err).Errorf("failed to register host <%s> to infra-env %s due to: %s",
 					params.NewHostParams.HostID, params.InfraEnvID.String(), err.Error())
 				// TODO Need event for infra-env instead of cluster
-				b.eventsHandler.AddEvent(ctx, params.InfraEnvID, params.NewHostParams.HostID, models.EventSeverityError,
-					err.Error(), time.Now())
+				// TODO: replace with proper InfraEnv event
```

Comment:
pls remove, it repeats of the previous comment line

Reply:
done